### DISCUSSION TITLE
How do I pass my client certs to generate client libraries if my backend API is backed by mutual TLS?

### DISCUSSION THREAD

**Discussion by balazzii** [Azure/autorest#4326] [Q&A] [answered]
Question:
My backend API is hosted in k8s and mutual TLS is enabled. I cannot access the APIs without client certificate.
Is there any argument in autorest to pass the client certificate to generate the client libraries?

**Comment by timotheeguerin** [comment, marked as answer]
Do you mean the spec is hosted behind authentication? If so we only support github private urls for now. You can always download the spec and run it locally.
